refocus-bdk-client.js

```diff
@@ -488,8 +488,11 @@ module.exports = (config) => {
               },
             };
           }
-
+          // adds name of action and an extra field set by bot developer
+          // for better logging granularity
+          const sumoLog = instance.body.name + instance.body.actionLog;
```

Comment:
@BrandonMurnane what happens if actionLog is undefined here? It's possible to leave actionLog as null when creating an event right? Correct me if I'm wrong but should this be something like:

const sumoLog = instance.body.name + instance.body.actionLog || '';